modules/ROOT/pages/create-api-specification-studio.adoc

```diff
@@ -0,0 +1,41 @@
+= Create an API Project in Studio
+
+Anypoint Studio 7.7 and later enables you to create a RAML or OAS 2.0 and 3.0 API specification, or an API fragment in Studio and automatically synchronize it with Design Center.
+
+It is not possible to create a flow out of an OAS 3.0 API specification.
+
+[NOTE]
+Studio uses the EGit plugin only for the VCS feature of editing API specifications offline. Studio does not support the EGit plugin for Mule application projects that you might track using your own VCS.
+
+To start creating an API project in Studio, you must be logged in to Anypoint Platform. See xref:xref:set-credentials-in-studio-to.adoc[Configure Platform Credentials].
+
+. In the taskbar at the top of the Anypoint Studio display, select *File* > *New* > *API Specificiation*.
+. Select the type of API project that you want to create.
+** To create a new API specification, select the *New API Specification* tab, and then select the type of API.
+** To create an API fragment, select the the *New API Fragment* tab, and then select the type of API fragment.
+. Type a name for your project.
+. Select *Finish*.
+
+Studio prompts you to switch to the API design perspective.
+
+In the API design perspective, use the API editor to write your API specification or API fragment. Additionally, you can use the API Console view to mock a live service so you can test your API specification.
+
+Studio uses VCS to track the development of your API project and keep it syncronized with Design Center.
+
+== Create Branches
+
+Using a different branch allows you to modify the API definition without risking pushing those changes to the main branch in Design Center.
+
+include::page$sync-api-projects-design-center.adoc[tag=create-branch-vcs]
+
+include::page$sync-api-projects-design-center.adoc[tag=commit-to-branch-vcs]
+
+== Merge and Push Your API Project
+
+You can merge the changes made to your API specification to your main branch and synchronize it with Design Center
```

Comment:
This is about changes made to the spec you CREATEd? 
Again, sounds like a separate topic:
CREATE a PROJECT (spec or frag)
MODIFY a PROJECT (branch and then merge changes)

Reply:
```suggestion
You can merge the changes made to your API specification to your main branch and synchronize it with Design Center:
```